app/uk/gov/hmrc/minorentityidentificationfrontend/api/controllers/JourneyController.scala

```diff
@@ -63,23 +66,30 @@ class JourneyController @Inject()(val authConnector: AuthConnector,
       authorised().retrieve(internalId) {
         case Some(authInternalId) =>
           journeyService.createJourney(req.body, authInternalId).map(
-            journeyId =>
+            journeyId => {
               businessEntity match {
                 case OverseasCompany => Created(Json.obj(
-                  "journeyStartUrl" -> s"${appConfig.selfUrl}${overseasControllerRoutes.CaptureUtrController.show(journeyId).url}"
+                  journeyStartUrl -> s"${appConfig.selfUrl}${overseasControllerRoutes.CaptureUtrController.show(journeyId).url}"
                 ))
                 case Trusts =>
                   auditService.auditJourney(journeyId, authInternalId)
+                  enable(TrustVerificationStub)
```

Comment:
We don't need this line. We'll enable/disable the FS using the test page or app config when needed